source/agora/consensus/Quorum.d

```diff
@@ -431,7 +483,8 @@ private NodeStake[] buildStakesDescending (const ref PublicKey filter,
 
 version (unittest)
 private QuorumConfig[PublicKey] buildTestQuorums (Range)(Range amounts,
-    const(PublicKey)[] keys, auto const ref Hash rand_seed)
+    const(PublicKey)[] keys, auto const ref Hash rand_seed,
+    auto const ref QuorumParams params)
```

Comment:
Why does DMD allow this ? T_T

Reply:
What's the issue? I want to be able to pass either lvalue or rvalue, but treat it as const.

Reply:
I was just surprised that could you stick the `const` between `auto` and `ref`, that's it.